client/src/js/components/reviewListEntry.jsx

```diff
@@ -1,9 +1,83 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { getFullMonth, makeStarElements, truncateWords } from '../../../../helpers/clientHelplers';
+import '../../css/reviewListEntry.css';
 
-const ReviewListEntry = () => (
-  <div>
-    ReviewListEntry
-  </div>
-);
+const starClassNames = {
+  containerClass: 'star-ratings user-rating',
+  fullStarClass: 'full-star user-rating',
+  pointFiveClass: 'point-five-star user-rating',
+  zeroStarClass: 'zero-star user-rating',
+  halfStarClass: 'half-star user-rating',
+  hiddenHalfStarClass: 'hidden-half-star user-rating',
+};
+
+class ReviewListEntry extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      textTruncated: this.props.review.text.length > 300,
+      truncatedText: (
+        (this.props.review.text.length > 300)
+          ? truncateWords(this.props.review.text)
+          : null
+      ),
+    };
+    this.date = new Date(this.props.review.date);
+  }
+
+
+  render() {
+    console.log('something2');
+    return (
+      <div className="reviewListEntry">
+        <div className="avatar">
+          <a href={this.props.review.avatar}>
+            <img
+              src={this.props.review.avatar}
+              height="48"
+              width="48"
+              alt={`${this.props.review.userName} User Profile`}
+              Title={`${this.props.review.userName} User Profile`}
+            />
+          </a>
+        </div>
+        <div className="name-and-date">
+          <span className="username">{this.props.review.userName}</span>
+          <br />
+          <span className="review-date">
+            {`${getFullMonth(this.date)} ${this.date.getFullYear()}`}
+          </span>
+        </div>
+        <div>{makeStarElements(this.props.review.aggregateRate / 5, 5, starClassNames)}</div>
+        <div className="review-text">{
+          this.state.textTruncated
+            ? (
+              <span>
+                {this.state.truncatedText}
+                <span className="clickable read-more">Read more</span>
+              </span>
+            )
+            : (
+              <span>
+                {this.props.review.text}
+              </span>
+          )
+        }
```

Comment:
this ternary matches the style guide, but just as a comment, to me example below makes more sense..
{ this.state.textTrundcated ?
    <>...</>
  ) : (
   <>...</>
  )
}